submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/model/ExtPageRequest.kt

```diff
@@ -1,3 +1,8 @@
 package ac.uk.ebi.biostd.submission.web.model
 
-class ExtPageRequest(val offset: Long, val limit: Int)
+class ExtPageRequest(
+    val fromRTime: String? = null,
```

Comment:
I think this one should be an Instant of OffsetDateTime

Reply:
Done